### PR TITLE
Creating stable version for settings type

### PR DESCRIPTION
**Pull request by chlahav**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [x] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [x] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by adxsdknet** [comment]
# Automation for azure-sdk-for-net 
A PR has been created for you: 
https://github.com/azure/azure-sdk-for-net/pull/-1 
.NET SDK Commits: 
https://github.com/adxsdknet/azure-sdk-for-net/pull/-1/commits/

File: specification/security/resource-manager/readme.md [hunk 1]
```diff
@@ -103,6 +103,35 @@ override-info:
   title: SecurityCenter
 ```
 
+### Tag: package-composite-v2
+
+These settings apply only when `--tag=package-composite-v3` is specified on the command line.
```
Comment (eliagrady) [inline specification/security/resource-manager/readme.md]: v3 in v2 section

File: specification/security/resource-manager/readme.md [hunk 2]
```diff
@@ -103,6 +103,35 @@ override-info:
   title: SecurityCenter
 ```
 
+### Tag: package-composite-v2
+
+These settings apply only when `--tag=package-composite-v3` is specified on the command line.
+
+```yaml $(tag) == 'package-composite-v2'
+input-file:
+- Microsoft.Security/stable/2018-06-01/pricings.json
```
Comment (eliagrady) [inline specification/security/resource-manager/readme.md]: I think we should sort by api-version date like [sql did here](https://github.com/Azure/azure-rest-api-specs/blob/master/specification/sql/resource-manager/readme.md)

**Review by eliagrady — CHANGES_REQUESTED** [review]
Minor changes

File: specification/security/resource-manager/readme.md [hunk 3]
```diff
@@ -103,6 +103,35 @@ override-info:
   title: SecurityCenter
 ```
 
+### Tag: package-composite-v2
```
Comment (sergey-shandar) [inline specification/security/resource-manager/readme.md]:
```suggestion
### Tag: package-composite-v3
```

**Review by sergey-shandar — CHANGES_REQUESTED** [review]

**Review by KrisBash — APPROVED** [review]
LGTM

**Review by eliagrady — APPROVED** [review]

**Review by sergey-shandar — APPROVED** [review]

**Pull request merged**